### DISCUSSION TITLE
Import box and selection areas are not correctly rendered

### DISCUSSION THREAD

**Discussion by opoupel** [dani-garcia/vaultwarden#5294] [Q&A] [answered]
Versions used:
Vaultwarden 1.32.6
Web-Vault 2024.6.2c

The box and selection area are not correctly rendered as shown on the following image

![vaultwadren_import-page](https://github.com/user-attachments/assets/ecbfc531-89f8-4bc7-b60f-b55db66efe58)

I have tested with different browser, Firefox, Chromium, Edge and I do not understand where the problem comes from, nor if I can solved it with configuration modification.

Thank you for your help.

PS: The avatar default icons are not rendered correctly neither but this is less important, or is it linked ?

**Comment by BlackDex** [comment, marked as answer]
Probably some security headers are added or overwritten. 
Please check via the Vaultwarden Backend if there are issues.
`/admin/diagnostics`

**Comment by opoupel** [comment]
Thank you !
My NGINX configuration had the following lines:

>     # Content Security Policy
>     add_header Content-Security-Policy "default-src 'self'; script-src 'self'";
>     # X-Frame-Options
>      add_header X-Frame-Options "SAMEORIGIN";

I removed them and it solved the issue.
